probe-rs/src/architecture/arm/sequences.rs

```diff
@@ -817,13 +817,28 @@ pub trait ArmDebugSequence: Send + Sync + Debug {
     ///
     /// [ARM SVD Debug Description]: https://open-cmsis-pack.github.io/Open-CMSIS-Pack-Spec/main/html/debug_description.html#debugPortStop
     #[doc(alias = "DebugPortStop")]
-    fn debug_port_stop(&self, interface: &mut dyn DapProbe) -> Result<(), ArmError> {
-        // Select Bank 0
-        interface.raw_write_register(PortType::DebugPort, Select::ADDRESS, 0)?;
-
-        // De-assert debug power request
-        interface.raw_write_register(PortType::DebugPort, Ctrl::ADDRESS, 0)?;
-
+    fn debug_port_stop(&self, _interface: &mut dyn DapProbe) -> Result<(), ArmError> {
+        // FIXME:
+        // We're currently not powering down the debug port because it causes issues with the RP2040
```

Comment:
Could we just add a custom sequence for the RP2040?

Reply:
maybe but I also found something else, that's why I put this back to draft for now